docs/install.md

```diff
@@ -53,6 +53,29 @@ To add the Tekton Pipelines component to an existing cluster:
    append the `--watch` flag to view the component's status updates in real
    time. Use CTRL + C to exit watch mode.
 
+### Installing Tekton Pipelines on OpenShift
+
+The `tekton-pipelines-controller` service account needs the `anyuid` security context constraint in order to run the webhook pod.
+
+_See [Security Context Constraints](https://docs.openshift.com/container-platform/3.11/admin_guide/manage_scc.html) for more information_
+
+1. Run the following commands to set up the project/namespace, and to install Tekton Pipelines.
+
+   ```bash
+   oc new project tekton-pipelines
+   oc adm policy add-scc-to-user anyuid -z tekton-pipelines-controller
```

Comment:
This commands requires admin privileges, is it better to add a note?
Or
Shall we add `oc login as admin` instruction as the first step?

Reply:
we also need the **hack** `cluster-admin` priv `oc adm policy add-cluster-role-to-user cluster-admin -z tekton-pipelines-controller -n tekton-pipelines` , don't we?

Reply:
In my testing, I only needed anyuid for the `tekton-pipelines-controller` sa.